packages/react/src/ReactChildren.js

```diff
@@ -28,6 +28,12 @@ var REACT_ELEMENT_TYPE =
 const REACT_PORTAL_TYPE =
   (typeof Symbol === 'function' && Symbol.for && Symbol.for('react.portal')) ||
   0xeaca;
+const REACT_CALL_TYPE =
```

Comment:
Minor nit: let's reorder them before the portal (the numbers go in sequential order).

Reply:
Makes sense.